tests/tests/File/Image/Svg/SanitizerTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Concrete\Tests\File\Image\Svg;
+
+use Concrete\Core\File\Image\Svg\Sanitizer;
+use PHPUnit_Framework_TestCase;
+
+class SanitizerTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var \Concrete\Core\File\Image\Svg\Sanitizer
+     */
+    protected static $sanitizer;
+
+    public static function setupBeforeClass()
+    {
+        parent::setUpBeforeClass();
+        self::$sanitizer = new Sanitizer();
```

Comment:
It looks like you're unintentially testing `SanitizerOptions` here as well because you are not providing a test double. From what I can tell, this code flow will cause a `new SanitizerOptions()` statement to run so changes to your SanitizerOptions implementation will break these tests. 

Instead you should have a test double options file provided with the various possible options and a separate test that unit tests the `SanitizeOptions` class

Reply:
I still contend that the options should be a test double and shouldn't be tested, but since it's just a value object I won't die on this hill :)